tests/ext/dd_trace_send_traces_via_thread.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Test the background sender does not leak
+--SKIPIF--
+<?php if (!getenv("DD_AGENT_HOST")) die("skip test if agent host is not set"); ?>
+--ENV--
+--FILE--
+<?php
+
+$host = getenv("DD_AGENT_HOST");
+$port = getenv("DD_TRACE_AGENT_PORT");
+if ($port === false) {
+    $port = '8126';
+}
+$url = "http://{$host}:{$port}/0.4/traces";
+$headers = [
+    'Datadog-Meta-Lang' => 'php',
+];
+
+// payload = [[]]
+$payload = "\x91\x90";
+
+
+dd_trace_send_traces_via_thread($url, $headers, $payload);
+
```

Comment:
Nit: Can we add an `echo 'Done';` here to ensure a `zend_bailout` didn't occur? :)